socorro/processor/hybrid_processor.py

```diff
@@ -20,15 +20,15 @@
 from configman import Namespace, RequiredConfig
 from configman.converters import class_converter
 
-from socorro.lib.datetimeutil import utc_now
+from socorrolib.lib.datetimeutil import utc_now
 from socorro.external.postgresql.dbapi2_util import (
     execute_query_fetchall,
 )
 from socorro.external.postgresql.connection_context import ConnectionContext
-from socorro.lib.transform_rules import TransformRuleSystem
-from socorro.lib.datetimeutil import datetimeFromISOdateString, UTC
-from socorro.lib.ooid import dateFromOoid
-from socorro.lib.util import (
+from socorrolib.lib.transform_rules import TransformRuleSystem
+from socorrolib.lib.datetimeutil import datetimeFromISOdateString, UTC
```

Comment:
We should at some point replace all of this with [arrow](http://crsmithdev.com/arrow/).